devito/core/cpu.py

```diff
@@ -139,7 +139,7 @@ def _specialize_clusters(cls, clusters, **kwargs):
         clusters = fuse(clusters)
 
         # Reduce flops
-        clusters = cse(clusters, sregistry)
+        clusters = cse(clusters, sregistry, cls.CSE_MIN_COST)
```

Comment:
I guess we say that if we need more options in the future we pass the whole `options` as in other optim passes?

Reply:
see explanation on slack:
https://devitocodes.slack.com/archives/C7JMLMSG0/p1663941583629559?thread_ts=1663929262.171029&cid=C7JMLMSG0